src/config.js

```diff
@@ -14,4 +14,6 @@ export default {
   enableReduxLogging: isDevelopment && !!global.btoa,
   enableSentry: !isDevelopment,
   sentryKey: 'ADD-DSN-HERE',
+  activePingInterval: 50 * 1000, // seconds between 2 presence fetch api call
```

Comment:
I was thinking we just want to issue activity when the user does something.
This is more complicated to do though.